src/api/app/views/webui/project/responsive_ux/_subprojects_actions.html.haml

```diff
@@ -0,0 +1,5 @@
+- content_for :actions do
+  - if User.possibly_nobody.can_modify?(project)
+    = link_to(new_project_path(namespace: project), class: 'nav-link') do
+      %i.fas.fa-plus-circle.fa-lg.mr-1
```

Comment:
`mr-1` should be `mr-2` to be consistent with other *actions partials.

Reply:
Oh well, I commented right when you pushed what I commented about :smile:

Reply:
Realized when I reviewed your PR :D